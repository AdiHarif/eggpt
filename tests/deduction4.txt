Mice are afraid of wolves.
Wolves are afraid of mice.
Cats are afraid of sheep.
Jessica is a cat.
Emily is a cat.
Winona is a mouse.
Sheep are afraid of wolves.
Gertrude is a sheep.
What is gertrude afraid of?
What is winona afraid of?
What is gertrude afraid of?
What is gertrude afraid of?