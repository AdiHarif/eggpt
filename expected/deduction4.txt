What is gertrude afraid of?	wolf	8 7
What is winona afraid of?	wolf	6 1
What is gertrude afraid of?	wolf	8 7
What is gertrude afraid of?	wolf	8 7